src/app/store/rubrics.reducer.ts

```diff
@@ -4,12 +4,12 @@ import { getMaxLevelsFromRubric } from '../functions/get-max-levels-from-rubric.
 import { getTotalPointsFromRubric } from '../functions/get-total-points-from-rubric.function';
 import { RubricInterface } from '../object-interfaces/rubric.interface';
 
-function _resetRubric(rubric: RubricInterface | undefined): RubricInterface | undefined {
-    return (rubric !== undefined) ? {
+function _resetRubric(rubric: RubricInterface): RubricInterface {
+    return {
         ...rubric,
         maxLevelsCount: getMaxLevelsFromRubric(rubric),
         totalPoints: getTotalPointsFromRubric(rubric),
-    } : rubric;
+    };
```

Comment:
I thought this change would be innocuous, but apparently it's not. When I created a rubric, I got an error because the rubric being passed in was undefined 😞